src/tokenize.js

```diff
@@ -548,13 +548,38 @@ pp.readString = function(quote) {
 
 // Reads template string tokens.
 
+const INVALID_TEMPLATE_ESCAPE_ERROR = {}
+
+pp.tryReadTemplateToken = function() {
```

Comment:
It looks like `nextToken` may return it: https://github.com/ternjs/acorn/blob/master/src/tokenize.js#L81